webapp/src/Twig/TwigExtension.php

```diff
@@ -189,10 +167,9 @@ public function printremainingminutes(float $start, float $end): string
 
     /**
      * Print a time formatted as specified. The format is according to date().
-     * @param string|float $datetime
      * @param Contest|null $contest If given, print time relative to that contest start.
      */
-    public function printtime($datetime, ?string $format = null, ?Contest $contest = null): string
+    public function printtime(string|float|null $datetime, ?string $format = null, ?Contest $contest = null): string
```

Comment:
```suggestion
    public function printtime(string|float ?$datetime, ?string $format = null, ?Contest $contest = null): string
```
@nickygerritsen  is this allowed?

Reply:
No this was never. What you mean is `?string|?float` but that is also not allowed